packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -107,6 +114,28 @@ export const Reports = () => {
     [reports, searchValue]
   );
 
+  useEffect(() => {
+    if (jobStarted) {
+      const interval = setInterval(async () => {
```

Comment:
Please use the window object to call these timer functions.

Reply:
Any reason why they should be called through the `window` object?

Reply:
It returns Node.JS types if you don't use the window object. Trying to stay clear of Node types is usually less risky due to some bundlers like Webpack 5 not supporting Node types by default any longer.

Reply:
`window` object is now used to call timer functions